API.md

```diff
@@ -32,6 +32,13 @@ LoRa.setPins(ss, reset, dio0);
 
 This call is optional and only needs to be used if you need to change the default pins used.
 
+### Saving pin (tested on ESP8266)
```

Comment:
Even though it's only been tested on ESP8266, I think we can leave out the text in the brackets.

Reply:
fine with me